quickstart/scaling_etcd.md

```diff
@@ -0,0 +1,230 @@
+---
+Title: Scaling up ETCD members
+---
+
+By default, Mayastor allows the creation of three ETCD members. If you wish to increase the number of etcd replicas, you will encounter an error. However, you can make the necessary configuration changes discussed in this guide to make it work.
+
+
+
+
+## Overview of StatefulSets
+
+
+StatefulSets are Kubernetes resources designed for managing stateful applications. They provide stable network identities and persistent storage for pods. StatefulSets ensure ordered deployment and scaling, support persistent volume claims, and manage the state of applications. They are commonly used for databases, messaging systems, and distributed file systems. Here's how StatefulSets function:
+* For a StatefulSet with N replicas, when pods are deployed, they are created sequentially in order from {0..N-1}.
+* When pods are deleted, they are terminated in reverse order from {N-1..0}.
+* Before a scaling operation is applied to a pod, all of its predecessors must be running and ready.
+* Before a pod is terminated, all of its successors must be completely shut down.
```

Comment:
The above content on generic STS is fine, but we need to add (link) to etcd in the messaging.